### PR TITLE
Adds the admin bar user ui

### PR DESCRIPTION
**Pull request by abea**

### CODE REVIEW

File: src/components/adminBar/TheAposAdminBar.vue [hunk 1]
```diff
@@ -34,20 +34,23 @@
...
 import AposContextMenu from '../contextMenu/AposContextMenu';
 
 export default {
   name: 'TheAposAdminBar',
   components: {
     AposLogo,
     AposButton,
-    AposContextMenu
+    AposContextMenu,
+    TheAposAdminBarUser
```
Comment (stuartromanek) [inline src/components/adminBar/TheAposAdminBar.vue:53]: what the what

File: src/components/adminBar/TheAposAdminBar.vue [hunk 2]
```diff
@@ -60,7 +63,8 @@ export default {
   data() {
     return {
       menuItems: [],
-      plusMenu: []
+      plusMenu: [],
```
Comment (stuartromanek) [inline src/components/adminBar/TheAposAdminBar.vue:66]: `create` or `new`

File: src/components/adminBar/TheAposAdminBar.vue (hunk 1 above)
Comment (abea) [reply src/components/adminBar/TheAposAdminBar.vue:53]: The `The` thing? Because it's only ever used once.

**Comment by stuartromanek** [comment]
image

the above screenshot not previewing for me, a new one ^

File: src/components/adminBar/TheAposAdminBar.vue (hunk 1 above)
Comment (stuartromanek) [reply src/components/adminBar/TheAposAdminBar.vue:53]: is that a real pattern? seems horrible for scanning dirs alphabetically

**Comment by abea** [comment]
@stuartromanek the subtle background color change doesn't seem like enough for the button focus state. An outline glow? Underline?

File: src/components/adminBar/TheAposAdminBar.vue (hunk 1 above)
Comment (abea) [reply src/components/adminBar/TheAposAdminBar.vue:53]: From here: https://vuejs.org/v2/style-guide/#Single-instance-component-names-strongly-recommended

**Pull request merged**